portfolio.html

```diff
@@ -0,0 +1,100 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>My Resume</title>
+
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+
+  <div class="container">
+
+    <nav>
+
+      <a href="index.html">Home</a>
+      <a href="#">About</a>
+      <a href="#">Portfolio</a>
+      <a href="#">Contact</a>
+
+    </nav>
+
+    <section class="portfolio">
+
+      <h1>Portfolio</h1>
+
+      <div class="portfolio-items">
+
+        <section class="portfolio-item">
+
+          <h2>Project Title</h2>
+
+          <img src="img/comingsoon.jpg" alt="coming soon">
+
+          <p>Project Description</p>
+
+          <a href="github.com">Github Link</a>
```

Comment:
Make these github anchor tags' `href` be `https://github.com`! Currently, the HTML is reading it as a URL in addition to yours (i.e., _www.yoursitedomain.com/github.com_).